src/controllers/users.js

```diff
@@ -11,3 +12,34 @@ exports.getUser = async (req, res, next) => {
   // This is an example of a successful response
   res.json({ success: true, data: user }); */
 };
+exports.updateProduct = async (req, res) => {
```

Comment:
Please use a validator here. Check [this](https://express-validator.github.io/)

Reply:
done you can check it, but do I need to do validation for every field in product model?